synapse/lib/httpapi.py

```diff
@@ -558,6 +558,9 @@ class AuthDelRoleV1(Handler):
 
 class ModelNormV1(Handler):
 
+    async def post(self):
+        await self.get()
```

Comment:
Lets make it return await self.get() just in case we eventually return something